writing-app/package-lock.json

```diff
@@ -0,0 +1,16577 @@
+{
```

Comment:
I don't think this file should be added into git. Otherwise, the PR looks good and all works well :)